homeassistant/components/profiler/__init__.py

```diff
@@ -136,13 +138,14 @@ async def _async_dump_scheduled(call: ServiceCall) -> None:
         original_maxother = arepr.maxother
         arepr.maxstring = 300
         arepr.maxother = 300
+        handle: asyncio.Handle
         try:
-            for handle in hass.loop._scheduled:  # pylint: disable=protected-access
+            for handle in getattr(hass.loop, "_scheduled"):
```

Comment:
`_scheduled` is not part of `AbstractEventLoop`, but it is assumed to be part of the implemented loop.
mypy complains with attr-defined so I think this is cleanest